assets/js/src/newsletters/automatic_emails/listings.jsx

```diff
@@ -178,7 +178,7 @@ class Listings extends React.Component {
       },
     }).done((response) => {
       if (response.data.status === 'active') {
-        MailPoet.Notice.success(MailPoet.I18n.t('automaticEmailActivated'));
+        MailPoet.Notice.success(MailPoet.I18n.t('automaticEmailActivated').replace('%1s', response.data.subject));
```

Comment:
To be consistent with other similar strings it should replace the placeholder with an automatic email group title (e.g. "First Purchase"), not an email subject. See an example here: https://github.com/mailpoet/mailpoet/blob/master/assets/js/src/newsletters/send/automatic.jsx#L47
```
const email = emails[newsletter.options.group];
...
{
  ...
  label: MailPoet.I18n.t('sendAutomaticEmailWhenHeading').replace('%1s', email.title),
  ...
}
```

Reply:
@wxa Sorry. You are right. I have changed my solution.